src/plugins/pdfPlayer/plugin.js

```diff
@@ -91,8 +115,12 @@ export class PdfPlayer {
     onWindowKeyDown(e) {
         if (!this.loaded) return;
 
-        // Skip modified keys
-        if (e.ctrlKey || e.altKey || e.metaKey || e.shiftKey) return;
+        // Skip modified keys for navigation/stop, allow for zoom (+/- usually need shift)
+        const isZoomKey = keyboardnavigation.getKeyName(e) === '+' || keyboardnavigation.getKeyName(e) === '=' || keyboardnavigation.getKeyName(e) === '-';
```

Comment:
Call `keyboardnavigation.getKeyName(e)` once.

Reply:
Apologies for the oversight, this should have been caught earlier. I likely missed it because I committed the changes after taking a break. You're right, the function shouldn’t have been called multiple times like that. I've corrected it now. Thanks for pointing it out!